.github/workflows/test-suite.yml

```diff
@@ -2,24 +2,47 @@ name: 'Test Suite'
 
 on:
   workflow_dispatch:
+    inputs:
+      terraform_matrix:
+        description: 'Terraform version matrix (JSON array)'
+        default: '["1.6.x", "1.5.x", "1.4.x", "1.3.x", "1.2.x"]'
+      provider_matrix:
+        description: 'MongoDB Atlas Provider version matrix (JSON array)'
+        default: '["", "1.12.3"]' # "" for latest version
+
   schedule:
     - cron: "0 0 * * *" # workflow runs every day at midnight UTC
```

Comment:
can we do longer frequency? every month?

Reply:
this frequency is for:
     - schedule_terraform_matrix: '["1.6.x", "1.0.8"]'
     - schedule_provider_matrix: '["", "1.12.3]' # "" for latest version  

the longer version test has to be run manually

Reply:
If we look at [the requirement](https://docs.google.com/document/d/1Z5WnGGdDKpae9azaqJvWZgqbZuMouRHCly8xzcarKAs/edit?disco=AAABCnhRna4) we discussed yesterday, we want to test our latest version against the terraform versions that we declare we support. This was also stated in one of [the document](https://docs.google.com/document/d/1YRorIH4UdW0yIxoSRuoZg-DP0dAmBhF3RDOp5O5Sb4o/edit#heading=h.v0fmnq3qwxa) Zuhair has recently proposed.

So my ask is: can we please have a monthly-based testing that checks **our latest provider version** against `'["1.6.x", "1.5.x", "1.4.x", "1.3.x", "1.2.x"]'`. I'd prefer not to do this manually as we have already lots of things we do manually. 

Let me know if I am missing something

Reply:
we could but the problem I see is that somebody has to look at the results, and if it's done once per month I see it very easy to miss that execution result.  Also I find it interesting to run it at other occasions like before a release.

Reply:
this is also happening with the daily Test Suite.  We run them automatically every night, but you have to go the GH UI to see the results.

Reply:
that said, I you still think it's a good idea please confirm and I'll add the monthly schedule

Reply:
On this, I don't have a strong opinion, I am ok to run it manually or before a release. @gssbzn  how do we do it for AtlasCLI?

Reply:
> how do we do it for AtlasCLI?

We always run packaging on master (we call it the waterfall on evergreen), this means after every merge, and then all the OS compatibility depends on packaging succeeding. The compatibility matrix exploded on the CLI at some point and evergreen let's us generate test programmatically so we write the yaml via code [here](https://github.com/mongodb/mongodb-atlas-cli/blob/master/tools/genevergreen/main.go)